rolling/distribution.yaml

```
@@ -2510,8 +2511,8 @@ repositories:
       - pilz_industrial_motion_planner_testutils
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/moveit2-release.git
-      version: 2.7.0-2
+      url: https://github.com/ros2-gbp/moveit2-release
```

```suggestion
      url: https://github.com/ros2-gbp/moveit2-release.git
```

This is an artifact of a configuration quirk that I'd like to track down but we prefer to keep the repository type suffix although it is not strictly necessary.